collector/collector_test.go

```diff
@@ -28,12 +28,14 @@ import (
 
 	"github.com/spacemonkeygo/monotime"
 	"github.com/stretchr/testify/assert"
-	"github.com/uber-go/zap"
 	"github.com/uber/arachne/config"
```

Comment:
`github.com/uber/arachne` are local includes that should be separate from the externals